docs/docs/Desenvolvimento técnico/endpoints.md

```diff
@@ -0,0 +1,2017 @@
+---
+title: Documentação dos Endpoints
+sidebar_position: 11
+description: Documentação dos endpoints do projeto.
+---
+
+# Documentação dos Endpoints da API
+
+## Método HTTP POST
+
+### Criar Usuário
+
+**Endpoint:** `/usuarios`  
+**Método HTTP:** `POST`  
+**Descrição:** Cria um novo usuário
+
+**Parâmetros do Corpo da Requisição:**
+
+- **email (obrigatório):** Email do usuário
+- **senha (obrigatório):** Senha do usuário
+- **tipo_usuario (obrigatório):** Tipo de usuário (ex: administrador)
+- **ativo (obrigatório):** Status do usuário (true para ativo, false para inativo)
+- **data_de_criacao (opcional):** Data e hora de criação do usuário (formato ISO 8601)
+
+**Exemplo de Requisição:**
+
+```http
+POST https://api.exemplo.com/v1/usuarios
+Content-Type: application/json
+
+{
+  "email": "usuario@example.com",
+  "senha": "senha123",
+  "tipo_usuario": "administrador",
```

Comment:
Usuário não pode ser adm.

Reply:
https://dev.to/adityaoberai/undestanding-role-based-access-control-with-aspnet-web-apis-1el8